protocol-units/bridge/integration-tests/tests/eth_movement.rs

```diff
@@ -59,7 +64,7 @@ async fn test_movement_client_build_and_fund_accounts() -> Result<(), anyhow::Er
 #[tokio::test]
 async fn test_movement_client_should_publish_package() -> Result<(), anyhow::Error> {
 	let _ = tracing_subscriber::fmt()
-        .with_max_level(tracing::Level::ERROR)
+        .with_max_level(tracing::Level::DEBUG)
```

Comment:
Let's not set the tracing level in the source code, as the output is quite noisy. Instead leave it up to the developer to set this env var if they want to debug.